sysutils/mac-brightnessctl/Portfile

```
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+PortGroup           github 1.0
```

Only nitpick might be maybe the `makefile` PortGroup could be useful here, that would eliminate the need for `use_configure no`.

```suggestion
PortGroup           github 1.0
PortGroup           makefile 1.0
```

---

Where can I learn more about this PortGroup? I don't see it documented at https://guide.macports.org/index.html#reference.portgroup.

---

Read the comments in https://github.com/macports/macports-ports/blob/master/_resources/port1.0/group/makefile-1.0.tcl and read other Portfiles that include the portgroup.

---

And it's not a nitpick. Whenever you use `use_configure no` with ports that create Mach-O binaries, you become responsible for manually implementing various MacPorts features, like [using the right compiler](https://trac.macports.org/wiki/UsingTheRightCompiler) and flags. The makefile portgroup implements it for you so you don't have to reinvent the wheel.

---

I've adopted the `makefile` PortGroup via https://github.com/macports/macports-ports/pull/27828/commits/58ede9398f9f8090cfa89664609ac0fe31ff24e4.